README.md

```diff
@@ -1 +1,37 @@
-# megafuel-js-sdk
\ No newline at end of file
+# megafuel-js-sdk
+
+This is Javascript SDK for the MegaFuel clients, it provides a simple way to interact with the MegaFuel.
+
+For more information, please refer to the [API documentation](https://docs.nodereal.io/docs/megafuel-api).
+
+## Quick Start
+1. Install dependency
+ ```shell
+ $ npm install
```

Comment:
this does not make sense to a js app. we should publish it to npm repository, and then developer config it in the `package.json`.

Reply:
done

Reply:
not fixed

Reply:
fixed now